docs/topics/mta-web-tagging-an-application.adoc

```diff
@@ -58,4 +58,46 @@ A side drawer opens.
 +
 You can see the tags attached to the application.
 +
-The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
\ No newline at end of file
+The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
+
+[id="creating-tags_{context}"]
+= Creating tags
+
+You can create custom tags for applications that {ProductShortName} migrates.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
+
++
+// Get updated image for MTA
+image::mta-web-assessment-tags-01.png[Tags]
++
+
+. Click *Create tag*, opening a dialog.
+. In the *Name* field enter a unique name for the tag.
+. Click the *Tag category* field and select a category tag to belong.
+. Click *Create*|.
+
+[id="editing-tags_{context}"]
+= Editing tags
+
+You can edit defined tags.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
+
++
+// Get updated image for MTA
+image::mta-web-assessment-tags-01.png[Tags]
++
+
+. Select a defined tag and click *Edit* to the right of the tag.
+. In the *Name* field enter a unique name for the tag, opening a dialog.
+. Edit the tag's name in the *Name* field.
+. Edit the *Rank* value.
+. Click the *Color* field and select a color for the tag. 
```

Comment:
@HagayVider1 this is the procedure to edit the tag category, to edit the tag - you need to expend the tag category row by clicking the arrow to the left -> Click the kebab menu to the right of the tag -> Click Edit

Reply:
See updated procedure and image lines 85-97